Dockerfile

```diff
@@ -17,9 +17,18 @@ RUN             git config --global user.email "email@example.com"
 RUN             virtualenv -p /usr/bin/python3 /venv
 RUN             . /venv/bin/activate && pip install jupyter jupytext matplotlib tensorflow torch
 
-RUN             wget https://julialang-s3.julialang.org/bin/linux/x64/1.3/julia-1.3.1-linux-x86_64.tar.gz
-RUN             tar -xzv < julia-1.3.1-linux-x86_64.tar.gz
-RUN             ln -s /julia-1.3.1/bin/julia /usr/bin/julia
+# Specify Julia version. Find current version on https://julialang.org/downloads/
+ARG             JULIA_VERSION_SHORT="1.5"
+ARG             JULIA_VERSION_FULL="${JULIA_VERSION_SHORT}.3"
+ENV             JULIA_INSTALLATION_PATH=/opt/julia
+
+RUN             wget https://julialang-s3.julialang.org/bin/linux/x64/${JULIA_VERSION_SHORT}/julia-${JULIA_VERSION_FULL}-linux-x86_64.tar.gz && \
+                tar zxf julia-${JULIA_VERSION_FULL}-linux-x86_64.tar.gz && \
+                mkdir -p "${JULIA_INSTALLATION_PATH}" && \
+                mv julia-${JULIA_VERSION_FULL} "${JULIA_INSTALLATION_PATH}/" && \
+                ln -fs "${JULIA_INSTALLATION_PATH}/julia-${JULIA_VERSION_FULL}/bin/julia" /usr/local/bin/ && \
```

Comment:
why `/usr/local/bin` rather than `/usr/bin`?  (answer might be "the convention is pretty vague anyway," just wondering)

Reply:
just because @fplk suggested it :)

Reply:
/usr/bin is for distribution-managed binaries.